tests/conftest.py

```
@@ -35,19 +35,19 @@ def cloned_template(tmp_path_factory):
+            if patch:
+                (git["apply", "--reject"] << patch)()
+                git("add", ".")
+                git(
+                    "commit",
+                    "--author=Test<test@test>",
+                    "--message=dirty changes",
+                    "--no-gpg-sign",
+                    "--no-verify",
+                )
+        git("tag", "--force", "v999999999.999999999.999999999")
+    yield dirty_template_clone
```

Ruff (0.15.21)

[warning] No teardown in fixture `cloned_template`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)